homeassistant/components/ebusd/__init__.py

```diff
@@ -59,14 +55,14 @@ def verify_ebusd_config(config):
 
 def setup(hass, config):
     """Set up the eBusd component."""
+    _LOGGER.debug("Ebusd integration setup started")
```

Comment:
You do not need to name the integration name in log messages, see our [style guidelines](https://developers.home-assistant.io/docs/development_guidelines#log-messages):
```suggestion
    _LOGGER.debug("Integration setup has started")
```

Reply:
I just moved the message, will do.

Reply:
Fixed in latest commit